edx_proctoring/templates/proctoring/seq_proctored_practice_exam_entrance.html

```diff
@@ -0,0 +1,70 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Would you like to take {{ display_name }} as a practice proctored exam?
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since this course is offered for credit, there will be one or more proctored exams in this course. Before you encounter a proctored exam that will count towards your grade and credit eligibility. You now have the option to take a practice proctored exam. Online proctoring is one requirement towards being eligible for credit.
```

Comment:
Do we want to include something like: "You can try this practice exam as many times as you want."